README.md

```diff
@@ -358,6 +358,8 @@ kubeclarity scan registry/nginx:private --config $HOME/own-kubeclarity-config
 Kubeclarity is using [k8schain](https://github.com/google/go-containerregistry/tree/main/pkg/authn/k8schain#k8schain) of google/go-containerregistry for authenticating to the registries.
 If the necessary service credentials are not discoverable by the k8schain, they can be defined via secrets described below.
 
+In addition, if service credentials are not located in "kubeclarity" Namespace, please set CREDS_SECRET_NAMESPACE to kubeclarity Deployment.
```

Comment:
Since the helm chart is setting `CREDS_SECRET_NAMESPACE` to the release namespace, I think that the comment should be in general for the creation of the secrets to be created in the release namespace and not in `kubeclarity` namespace as described in the examples below.

Reply:
@FrimIdan Thank you for review. I appended case of using helm charts.